$OpenBSD: patch-psutil_tests___init___py,v 1.1 2016/04/26 13:03:40 landry Exp $
the scripts/ subdir is only present in the tarball, not on the system
--- psutil/tests/__init__.py.orig	Tue Apr 26 14:57:59 2016
+++ psutil/tests/__init__.py	Tue Apr 26 14:58:48 2016
@@ -91,7 +91,7 @@ if not PY3:
 
 ROOT_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__),
                            '..', '..'))
-SCRIPTS_DIR = os.path.join(ROOT_DIR, 'scripts')
+SCRIPTS_DIR = 'scripts'
 
 WIN_VISTA = (6, 0, 0) if WINDOWS else None
 VALID_PROC_STATUSES = [getattr(psutil, x) for x in dir(psutil)
